drivers/openstack/openstack.go

```diff
@@ -384,14 +384,15 @@ func (d *Driver) Stop() error {
 }
 
 func (d *Driver) Remove() error {
-	log.WithField("MachineId", d.MachineId).Info("Deleting OpenStack instance...")
+	log.WithField("MachineId", d.MachineId).Debug("deleting instance...")
+	log.Info("Deleting OpenStack instance...")
```

Comment:
I think so.  This way you can see the Machine ID if you enable debug.  What do you think?  Mainly I just wanted to get the same feel across drivers for the `Info`.